geo-distributed-deployment-topology.md

```diff
@@ -57,8 +57,8 @@ This section describes the key parameter configuration of the TiDB geo-distribut
 - To prevent remote TiKV nodes from launching unnecessary Raft elections, it is required to increase the minimum and maximum number of ticks that the remote TiKV nodes need to launch an election. The two parameters are set to `0` by default.
 
     ```yaml
-    raftstore.raft-min-election-timeout-ticks: 1000
-    raftstore.raft-max-election-timeout-ticks: 1020
+    raftstore.raft-min-election-timeout-ticks: 50
+    raftstore.raft-max-election-timeout-ticks: 60
     ```
 
```

Comment:
```suggestion
> **Note:**
>
> Configuring larger values for `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` can greatly reduce the probability that a peer on this TiKV will become the leader. However, in a disaster scenario, if some TiKV nodes are down and the active TiKV node is lagging behind in Raft logs, only the Region on this TiKV can become the leader. When the leader is down, the Region on this TiKV node needs to wait for at least the time set by `raftstore.raft-min-election-timeout-ticks' to become the leader. It is recommended that you do not set these values too large to avoid affecting the availability of the cluster in this scenario.
```